imdown-mobile/login/login.tsx

```diff
@@ -50,6 +50,7 @@ const Login = ({ navigation }) => {
           email: resp.user.email
         }
         signInOnBackend(user)
+        goToSquads()
```

Comment:
Let's actually move this into the `signInOnBackend()` function and replace the `console.log` with this. We only want to go to the squad page if we successfully sign in on the backend. Right now, `signInOnBackend` is async, so we can transition to the squad page before we authenticate with our backend which we don't want. To enforce that we have signed in successfully, we should only go to the squad page in the final `then` call in the Promise chain in `signInOnBackend`